mne/channels/channels.py

```diff
@@ -613,7 +618,10 @@ def add_channels(self, add_list, copy=None):
 
         # Create final data / info objects
         data = np.concatenate(data, axis=con_axis)
-        infos = [self.info] + [inst.info for inst in add_list]
+        infos_to_add = deepcopy([inst.info for inst in add_list])
+        if force_update_info is True:
+            _force_update_info(self.info, infos_to_add)
+        infos = [self.info] + infos_to_add
         new_info = _merge_info(infos)
```

Comment:
I thought about doing it that way but opted not just to keep from complicating that function more, but I'm happy to move this stuff in there. I can't really think of a case where you'd want to do the field updating *without* subsequently merging. It'd also remove an extra function...